src/pages/Admin/MapPreview/MapPreview.scss

```diff
@@ -0,0 +1,20 @@
+@import "scss/constants.scss";
+
+.MapPreview {
+  width: 100%;
+  margin-bottom: 1rem;
```

Comment:
Usually `1rem` === `16px` i.e. `$spacing--lg`.

`rem` makes sense when putting space between text and other objects.

There might be need of something in the vein of
 ```scss
@function size-factor($multiplier) {
  // NOTE: $spacing--lg could be used because 16px = 1rem (base) font-size
  @return 1rem * $multiplier;
}
```
or 

```scss
// Relative font sizes
$spacing--relative-to-doc: 1rem;
$spacing--relative-to-parent: 1em;
```

in `scss/constants.scss` for this and similar uses.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2006/commits/0c72b4d0e7a6245b75d442138529cfd256342c4b